cli/tools/repl/editor.rs

```diff
@@ -372,6 +372,7 @@ pub struct ReplEditor {
   inner: Arc<Mutex<Editor<EditorHelper>>>,
   history_file_path: PathBuf,
   errored_on_history_save: Arc<AtomicBool>,
+  pub should_exit_on_interrupt: Arc<AtomicBool>,
```

Comment:
Nitpick: I think it would be good to make this private and then add some methods for getting and setting. That way the type of this field can be the private details of this struct (it's not necessary for the code in repl/mod.rs to know this is an atomic bool) and the `Ordering` used is isolated to this file and not spread around multiple files.